assets/templates/login.html

```diff
@@ -6,6 +6,7 @@
     <title>Cozy</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="{{asset .Domain "/fonts/fonts.css"}}">
+    <link rel="stylesheet" href="{{asset .Domain "/styles/cozy-ui.css"}}">
```

Comment:
What is the `/styles/cozy-ui.css` file?

Reply:
oh sorry, it's was for working with it locally. I'll remove it.